presidio-analyzer/presidio_analyzer/recognizer_registry/recognizer_registry.py

```diff
@@ -69,46 +71,51 @@ def __init__(
         self,
         recognizers: Optional[Iterable[EntityRecognizer]] = None,
         global_regex_flags: Optional[int] = re.DOTALL | re.MULTILINE | re.IGNORECASE,
-        supported_languages: Optional[List[str]] = None
+        supported_languages: Optional[List[str]] = None,
     ):
         if recognizers:
             self.recognizers = recognizers
         else:
             self.recognizers = []
         self.global_regex_flags = global_regex_flags
-        self.supported_languages = supported_languages
+        self.supported_languages = (
+            supported_languages if supported_languages else ["en"]
+        )
 
-    def _create_nlp_recognizer(self,
-                               nlp_engine: NlpEngine = None,
-                               supported_language: str = None) -> SpacyRecognizer:
+    def _create_nlp_recognizer(
+        self, nlp_engine: NlpEngine = None, supported_language: str = None
+    ) -> SpacyRecognizer:
         nlp_recognizer = self._get_nlp_recognizer(nlp_engine)
 
         if nlp_engine:
             return nlp_recognizer(
                 supported_language=supported_language,
                 supported_entities=nlp_engine.get_supported_entities(),
-                )
+            )
 
         return nlp_recognizer(supported_language=supported_language)
 
-    def add_nlp_recognizer(self, nlp_engine: NlpEngine = None) -> None:
+    def add_nlp_recognizer(self, nlp_engine: NlpEngine) -> None:
```

Comment:
nlp_engine is mandatory to get the languages and the list of entities right.

Reply:
That was the previous implementation of load predefined recognizers, I didn't want to break it